config/routes.rb

```diff
@@ -28,6 +28,10 @@
   get 'intent', to: 'intents#show'
   get 'custom.css', to: 'custom_css#show', as: :custom_css
 
+  resource :instance_actor, path: 'instance-actor', only: [:show] do
```

Comment:
Maybe just `actor`? There's no overlap either way. Seems redundant.

Reply:
For the controller name? The path? I thought having “instance” in the name would make it clearer that it is meant to represent the instance itself.

Reply:
For the path! Seems cleaner that way for two reasons: Our ActivityPub JSON doesn't use the word "instance" anywhere, and other multi-word URLs use underscores instead of dashes, so there would be a consistency issue.

Reply:
Ok, changed.